Assignments/Task-2/WordCount.txt

```diff
@@ -0,0 +1,13 @@
+//https://www.codewars.com/kata/570cc83df616a85944001315/train/javascript
+
+function countWords(str) {
+var result;
+  var resultString = str.match(/\S+/g);
+  if(resultString != null) {
+    result = resultString.length;
+  } else {
+     result = 0;
+   } 
+  return result;
```

Comment:
Please verify below test cases:
it("should work with non-whitespace (ex. breakspace) chars", function() {
    Test.assertEquals(countWords("﻿Hello﻿World "), 2);
    Test.assertEquals(countWords("Hello﻿World"), 2);
  });

If fails, update the logic

Reply:
These test cases are getting passed when i am executing the code with the current logic only


![Screenshot (3)](https://user-images.githubusercontent.com/61185384/75872047-83744680-5e33-11ea-9dc0-82ccbda47198.png)